packages/dev-env/src/network.ts

```diff
@@ -73,7 +82,7 @@ export class TestNetwork extends TestNetworkNoAppView {
       plcUrl: plc.url,
       signingKey: ozoneKey,
       serverDid: ozoneDid,
-      dbPostgresSchema: `ozone_${dbPostgresSchema}`,
+      dbPostgresSchema: `ozone_${dbPostgresSchema || 'db'}`,
```

Comment:
unrelated change but this prevents the local db from being named `ozone_undefined`